tools/sizediff

```diff
@@ -5,26 +5,16 @@
 import sys
```

Comment:
@ysoldak perhaps you could please put the changes to `sizediff` into a separate commit/PR?